tests/core/validation/test_outbound_validation.py

```diff
@@ -14,307 +16,452 @@
         ),
-        (
-            _make_dynamic_fee_txn(
+        pytest.param(
+            make_blob_txn(
                 access_list=((b"\xef" * 20, (1, 2, 3, 4)),),
             ),
             True,
+            id="valid_blob_with_storage_keys",
+        ),
+        pytest.param(
+            make_access_list_txn(access_list=()),
+            True,
+            id="valid_empty_access_list",
+        ),
+        pytest.param(
+            make_dynamic_fee_txn(access_list=()),
+            True,
+            id="valid_empty_dynamic_fee_access_list",
         ),
-        (_make_access_list_txn(access_list=()), True),
-        (_make_dynamic_fee_txn(access_list=()), True),
-        (
-            _make_access_list_txn(
+        pytest.param(
+            make_blob_txn(access_list=()),
+            True,
+            id="valid_empty_blob_access_list",
+        ),
+        pytest.param(
+            make_access_list_txn(
                 access_list=((b"\xf0" * 19, (0, 2)),),
             ),
             False,
+            id="invalid_short_address_in_access_list",
         ),
-        (
-            _make_dynamic_fee_txn(
+        pytest.param(
+            make_dynamic_fee_txn(
                 access_list=((b"\xf0" * 19, ()),),
             ),
             False,
+            id="invalid_short_address_with_empty_storage_keys",
         ),
-        (
-            _make_access_list_txn(
+        pytest.param(
+            make_blob_txn(
+                access_list=((b"\xf0" * 19, ()),),
+            ),
+            False,
+            id="invalid_short_address_in_blob_access_list",
+        ),
+        pytest.param(
+            make_access_list_txn(
                 access_list=((b"\xf0" * 20, ("0", 2)),),
             ),
             False,
+            id="invalid_string_storage_key",
         ),
-        (
-            _make_dynamic_fee_txn(
+        pytest.param(
+            make_dynamic_fee_txn(
                 access_list=((b"\xf0" * 20, (b"", 1)),),
             ),
             False,
+            id="invalid_empty_bytes_storage_key",
         ),
-        (
-            _make_dynamic_fee_txn(
+        pytest.param(
+            make_blob_txn(
+                access_list=((b"\xf0" * 20, (b"", 1)),),
+            ),
+            False,
+            id="invalid_empty_bytes_storage_key_in_blob",
+        ),
+        pytest.param(
+            make_dynamic_fee_txn(
+                access_list=(("", (1, 2)),),
+            ),
+            False,
+            id="invalid_empty_string_address_in_access_list",
+        ),
+        pytest.param(
+            make_blob_txn(
                 access_list=(("", (1, 2)),),
             ),
             False,
+            id="invalid_empty_string_address_in_blob_access_list",
+        ),
+        pytest.param({}, False, id="invalid_empty_dict"),
+        pytest.param(
+            merge(make_legacy_txn(), {"invalid-key": 1}),
+            False,
+            id="invalid_extra_key",
```

Comment:
Thank you for all of the ids here ⭐ that makes it much easier to see what each is testing!